python/examples/matmul/test_schedule_dialect.py

```diff
@@ -0,0 +1,69 @@
+# RUN: %PYTHON %s 2>&1 | FileCheck %s
+
+# This file contains small benchmarks with reasonably-sized problem/tiling sizes
+# and codegen options.
+
+import mlir.iree_sandbox as sandbox
+import mlir.ir as ir
+import mlir.dialects.pdl as pdl
+import mlir.dialects.linalg_transform as transform
+
+from ..core.experts import *
+from ..core.harness import *
+from ..core.transforms import *
+
+from ..contraction.definitions import EinsumProblem
+
+################################################################################
+### Problem instantiations.
+################################################################################
+
+keys = ['m', 'n', 'k']
+
+# CHECK-NOT: FAILURE
+
+
+def add_tile_schedule(module):
+  with InsertionPoint(module.body):
+    pdl_pattern_name = 'pdl_pattern_name'
+    pdl_pattern = pdl.PatternOp(\
+      1, #=benefit 
+      pdl_pattern_name#=sym_name
```

Comment:
Nit: this is Python, so we should be able to write `benefit=1, sym_name=pdl_pattern_name`.

Reply:
interestingly, this complains about unexpected `sym_name` which makes me think we are not calling the function I am expecting .. could you patch and try on your end?

Reply:
sigh .. thanks!